<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainW</class>
 <widget class="QWidget" name="mainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>mainW</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainW
{
	border-image: url(:/new/prefix1/image/mainimag/board_img.png);
}</string>
  </property>
  <widget class="QPushButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>294</y>
     <width>250</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/mainimag/tool.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>250</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/mainimag/clearn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="moreButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>548</y>
     <width>250</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/mainimag/more.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>40</y>
     <width>714</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox
{
	border-image: url(:/new/prefix1/image/mainimag/back_board1.png);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="schedule_label">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>20</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scheVal_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="ele_label">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>20</y>
      <width>30</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="eleVal_label">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>20</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="water_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>13</y>
      <width>44</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/mainimag/water.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>20</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/mainimag/time.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusIcon_label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>25</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/mainimag/status.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="waterVal_label">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>20</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeVal_label">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>20</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>20</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>已到达加水点</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>toolButton</zorder>
  <zorder>cleanButton</zorder>
  <zorder>moreButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
